src/EventHandler/Message.php

```diff
@@ -259,6 +261,78 @@ public function delReaction(int|string $reaction): array
         return $this->reactions;
     }
 
+    /**
+     * Translate text message(for media translate it caption).
+     *
+     * @param string $toLang Two-letter ISO 639-1 language code of the language to which the message is translated
+     *
+     */
+    public function translate(
+        string $toLang = 'en'
+    ): string {
+        Assert::notEmpty($this->message);
```

Comment:
Just return the message itself if it's empty

Reply:
error while i using `$MadelineProto = new API('session.madeline');`

Reply:
its ok i've also forgot to write phpdoc for CallbackQuery class methods ... will fix

Reply:
@error1001es  please open an issue with full details (your code and MadelineProto.log file) 
this is not a good place to say your issue!!